gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceTest.java

```diff
@@ -225,6 +226,42 @@ public void removeUserProjectFail() throws URISyntaxException {
         service.removeUserFromProject(project, account);
     }
 
+    @Test(expectedExceptions = GoodDataException.class)
```

Comment:
Could you please add `expectedExceptionsMessageRegExp` to `@Test` annotations so it's verified also that the correct message is returned to user?

Reply:
added